addons/nuxeo-wopi/nuxeo-wopi/src/main/java/org/nuxeo/wopi/Helpers.java

```diff
@@ -100,20 +100,20 @@ public static Blob getEditableBlob(DocumentModel doc, String xpath) {
                     () -> xpath);
             return null;
         }
-        // ignore external blob providers
-        if (isExternalBlobProvider(blob)) {
+        // ignore blob providers that don't support sync
+        if (!supportsSync(blob)) {
             log.debug(
-                    "Blobs: repository={} docId={} xpath={} Ignoring blob as it is backed by a BlobProvider preventing updates",
+                    "Blobs: repository={} docId={} xpath={} Ignoring blob as it is backed by a BlobProvider preventing sync",
                     doc::getRepositoryName, doc::getId, () -> xpath);
             return null;
         }
         return blob;
     }
 
-    protected static boolean isExternalBlobProvider(Blob blob) {
```

Comment:
Isn't it better to keep it as deprecated in a HF?
I know it's probably not used by any customer, but well, let's be safe.

Reply:
It's `protected` so a customer would really have to go through contortions to be able to use it... I'm willing to take the risk here.

Reply:
Well, I do not agree. We should always keep and deprecate public/protected stuff on the maintenance branches, that's the "rule".

Do as you wish.

Reply:
Ok you're right, updated.